docs/customization/subplots.ipynb

```diff
@@ -0,0 +1,148 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "32fb725d-602c-433b-8522-bd8be63fa4d0",
+   "metadata": {},
+   "source": [
+    "# Subplots / Tiled plots\n",
+    "\n",
+    "There are two ways to create subplot layouts with Plopp:\n",
+    "\n",
+    "- Using manually created Matplotlib axes\n",
+    "- Placing figures inside container widgets\n",
+    "\n",
+    "The first method will work with any (static or interactive) backend,\n",
+    "while the second will only work with the interactive (`%matplotlib widget`) backend.\n",
+    "\n",
+    "## Using Matplotlib axes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "40c4c95a-0b3c-4603-be82-f1555bba07bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import plopp as pp\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "da = pp.data.data_array(ndim=3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3b377e93-f7a5-4903-9bb2-a90c1844c3c7",
+   "metadata": {},
+   "source": [
+    "To create our subplots, we use Matplotlib's in-built `plt.subplots` function.\n",
+    "We then pass each subplot axes to Plopp's `plot` function as the `ax` argument:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "46d49d86-b72b-4083-98ff-6c322c1badca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots(2, 2, figsize=(8, 6))\n",
+    "\n",
+    "p1 = pp.plot(da['zz', 0], ax=ax[0, 0])\n",
+    "p2 = pp.plot(da['zz', 0]['yy', 0], ax=ax[0, 1])\n",
+    "p3 = pp.plot(da['xx', 0]['yy', 0], ax=ax[1, 0])\n",
+    "p4 = pp.plot(da['xx', -1], cmap='magma', ax=ax[1, 1])\n",
+    "\n",
+    "fig.tight_layout()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4af0079c-bf62-4a06-b0f0-109e2acc0483",
+   "metadata": {},
+   "source": [
+    "## Using container widgets\n",
+    "\n",
+    "When the interactive backend is in use,\n",
+    "each figure is a widget that can be placed inside other `ipywidgets` for controlling the layout.\n",
+    "\n",
+    "We however note that in this case, alignment of axes is not ensured.\n",
+    "This method for creating subplots is applicable for quickly creating interactive interfaces,\n",
+    "but we do not recommend it for publication purposes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c112ba21-7fdc-462c-911f-c768b2d70aa0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib widget\n",
+    "import ipywidgets as ipw"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "61f0a00d-fedf-42fb-bf92-303f826dad30",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "size = (4, 3)\n",
+    "\n",
+    "p1 = pp.plot(da['zz', 0], figsize=size)\n",
```

Comment:
I was confused here for a couple of seconds, thinking that the `figsize=size` argument took the role of the `ax` argument, telling plopp the "size" of the layout.

Can we avoid specifying the `figsize`? It distracts from what actually needs to be done? Or add an example with 2 plots first, and then give this, explaining how we may want/need to scale down the individual plots?

Reply:
What about doing
```Py
subplot_size = (4, 3)
p1 = pp.plot(da['zz', 0], figsize=subplot_size)
```
?

Reply:
Or just split the cell? The creation of the plots is really not the main point, so write something like:

```
Given a numbers of plots, p1, ...:
<code for making plots>
we can create a figure with subplots using:
<VBox/HBox code, etc.>
```

Reply:
I'll do both ;-)